src/command/http-command.ts

```diff
@@ -324,7 +324,10 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 	}
 
 	private parseStreamTimings (stream: Request) {
-		const timings = { ...{ end: Date.now(), phases: {} }, ...stream.timings };
+		const timings = { ...stream.timings };
+		timings.end = timings.end ?? Date.now();
```

Comment:
@alexey-yarmosh do we know the underlying reason why the `end` might be undefined while some other fields are still present?

Reply:
I’ve opened an issue, so if there is any update we will know
https://github.com/sindresorhus/got/issues/2282